background.ts

```diff
@@ -49,6 +49,7 @@ const k8smanager = newK8sManager();
 const diagnostics: DiagnosticsManager = new DiagnosticsManager();
 
 let cfg: settings.Settings;
+let inFirstRun = false;
```

Comment:
Shouldn't this default to `true` on a first-run so you don't have a race condition?

```suggestion
let inFirstRun = settings.isFirstRun();
```

Reply:
Disagree. Those are two different things. `settings.isFirstRun()` means that we need to put up the first-run dialog. `inFirstRun` is true when the first-run window is up.

Reply:
So how do you prevent the main window from being raised in the time after the app is launched, but before the first-run dialog is displayed? E.g. if somebody tripple-clicks on the dock icon to start the app.